rust/agents/scraper/migration/src/l20220805_types.rs

```diff
@@ -1,9 +1,11 @@
 use sea_orm_migration::prelude::*;
 
+/// Hashes are to be stored as lowercase hex chars without a 0x prefix
 #[allow(non_upper_case_globals)]
-pub const Hash: ColumnType = ColumnType::Binary(BlobSize::Blob(Some(256 / 8)));
+pub const Hash: ColumnType = ColumnType::String(Some(64));
+/// Addresses are to be stored as lowercase hex chars without a 0x prefix
```

Comment:
at the risk of being pedantic just wanna flag that at some point these (& Hash up above) may not always be strictly hex chars if we support a chain like Solana where hashes/addresses are not necessarily hex chars. Obviously String is compatible with this so no need to change anything

Reply:
For what it is worth, those strings are varchars with a limit of 64, so it is very very easy to increase the limit later. Since the data is so structured right now it makes sense to set it as a sanity check IMO.